packages/runtime/test-runtime-utils/src/mocksForReconnection.ts

```diff
@@ -33,15 +33,17 @@ export class MockContainerRuntimeForReconnection extends MockContainerRuntime {
             this.clientId = uuid();
             // Update the clientId in FluidDataStoreRuntime.
             this.dataStoreRuntime.clientId = this.clientId;
+            this.factory.quorum.addMember(this.clientId, {});
```

Comment:
This is still probably not sufficient -- with a shared quorum, offline clients will still see live updates to the quorum.

Reply:
Not planning to improve in this PR, tracking this improvement in AB#535